### PR TITLE
Add "mini-omni2" as a library

### PR DESCRIPTION
**Pull request by NielsRogge**
This PR makes sure download stats work for https://huggingface.co/gpt-omni/mini-omni2.

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts [hunk 1]
```diff
@@ -373,6 +373,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/abetlen/llama-cpp-python",
 		snippets: snippets.llama_cpp_python,
 	},
+	"mini-omni2": {
+		prettyLabel: "Mini-Omni2",
+		repoName: "mini-omni2",
+		repoUrl: "https://github.com/gpt-omni/mini-omni2",
+		countDownloads: `path_extension:"pt" OR path_extension:"pth"`,
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:380]:
```suggestion
		countDownloads: `path:"model_config.yaml"`,
```

Shouldn't we count downloads of `model_config.yaml` for instance to be sure we don't double-count things? From what I can see, the full repo is download once when using the library ([here](https://github.com/gpt-omni/mini-omni2/blob/9289ddb69c05d39b4fa6f353225facd00bdde400/inference.py#L376)).

**Review by Vaibhavs10 — APPROVED** [review]
LGTM, the repo also has a tag 👍

File: packages/tasks/src/model-libraries.ts [hunk 2]
```diff
@@ -373,6 +373,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/abetlen/llama-cpp-python",
 		snippets: snippets.llama_cpp_python,
 	},
+	"mini-omni2": {
+		prettyLabel: "Mini-Omni2",
+		repoName: "mini-omni2",
```
Comment (Wauplin) [inline packages/tasks/src/model-libraries.ts:378]:
```suggestion
		repoName: "Mini-Omni2",
```

**Review by Wauplin — APPROVED** [review]
Thanks @NielsRogge !

**Pull request merged**